### PR TITLE
Fix demo

### PR DESCRIPTION
**Pull request by buzzware**
This contains a few fixes to the demo, and works on my OSX machine, but with Windows Server 2019. Perhaps it should have 2019 as an additional target, and a 2016 image that works should be substituted

### CODE REVIEW

**Comment by strokirk** [comment]
I can confirm that these changes fixes the setup process!

File: Vagrantfile [hunk 1]
```diff
@@ -10,7 +12,7 @@ Vagrant.configure("2") do |config|
     end
 
     # If you change this IP, also change the DNS server for the "web" VM.
-    adfs2016.vm.network "private_network", ip: "10.0.0.2"
+    adfs2016.vm.network "private_network", ip: "11.0.0.2"
```
Comment (strokirk) [inline Vagrantfile:15]: Note that this IP is not private (unlike the 10.0.0.0/8 block). Any reason why we can't use an adress in a private range?

Comment (buzzware) [reply Vagrantfile:15]: No, perhaps 10.1.* then. I was avoiding what was already being used in my scenario, and can be a wifi router range

**Comment by jobec** [comment]
See a942f9a for fixes for this.
I switched the entire setup to 2019 server, removing the server 2012 example.

**Pull request closed without merging**